src/printer.js

```diff
@@ -2472,7 +2472,15 @@ function genericPrintNoParens(path, options, print, args) {
       } else if (n.body) {
         parts.push(
           " {",
-          indent(concat([line, group(path.call(print, "body"))])),
+          indent(concat([
```

Comment:
this is going to fail because of the prettier lint rule. You now need to do `yarn lint -- --fix` before committing :p